package.json

```diff
@@ -134,9 +134,10 @@
     "moment": "2.17.1",
     "rc-calendar": "7.5.1",
     "rc-time-picker": "2.2.1",
+    "react": "15.4.1",
     "react-addons-css-transition-group": "15.4.2",
     "react-dom": "15.4.1",
-    "react": "15.4.1",
+    "rtl-detect": "1.0.0",
```

Comment:
That's a _lot_ of code to pull in for something that should be a simple check against an array, or css selector `:-moz-locale-dir(rtl)`…